llvm/tools/llvm-exegesis/lib/CodeTemplate.h

```diff
@@ -35,6 +35,8 @@ struct InstructionTemplate {
   const MCOperand &getValueFor(const Variable &Var) const;
   MCOperand &getValueFor(const Operand &Op);
   const MCOperand &getValueFor(const Operand &Op) const;
+  MCOperand &getValueFor(unsigned OpIdx);
```

Comment:
Having the same name for this overload is confusing: the two overloads for `Variable` and `Operand` are unambiguous thanks to the types, but this one is unclear as to whether we're talking about an operand or variable.  `getValueForOperandIdx` ?

Reply:
Addressed